examples/apps/collaborative-textarea/package.json

```diff
@@ -78,17 +78,18 @@
     "eslint-plugin-react": "~7.28.0",
     "eslint-plugin-tsdoc": "~0.2.14",
     "eslint-plugin-unicorn": "~40.0.0",
-    "html-webpack-plugin": "^4.5.2",
+    "html-webpack-plugin": "^5.5.0",
     "jest": "^26.6.3",
     "jest-junit": "^10.0.0",
     "jest-puppeteer": "^4.3.0",
+    "process": "^0.11.10",
```

Comment:
I'd like to track down why this polyfill is needed, so I can update #9508 if needed. Do you know where it's coming from?

I think our goal should be that our packages are usable in Webpack _without_ adding polyfills to your config. As of now the plan is to disallow node polyfills except for `events` and `url`, but that goal is really focused on our libs. Examples _could_ continue to use polyfills directly in Webpack like you do here, but that implies that our customers will too.

Reply:
Oh, I see now in a comment later:

> We do however transitively depend on the `util` npm package (node_modules/util/util.js) which requires `process.env` to be defined. <https://github.com/browserify/node-util/issues/57#issuecomment-764436352>

I'll file an issue if you haven't already. At the very least this will need to be documented when using our packages with Webpack 5.

Reply:
I have not filed any issues about this. To me this looks like a bug in util (has an undeclared dependency), and a quality issue with whatever we use that depends on util (which would be nice for it not to depend on nodejs stuff). I'll leave how to track/address this to you.

When using webpack, I generally expect random transitive dependencies to need manual fiddling (it makes me sad this is something I have accepted), but I suppose trying to get this fixed could help our consumers and may be worth it.